projects/vdk-plugins/vdk-impala/tests/functional/template_regression_test.py

```diff
@@ -108,6 +109,56 @@ def test_load_dimension_scd1_bad_target_schema(self) -> None:
             template_args=template_args,
         )
 
+    def test_load_dimension_scd1_checks_positive(self) -> None:
+
+        test_schema = "vdkprototypes"
+        source_view = "vw_dim_org"
+        target_table = "dw_dim_org"
+        staging_schema = "staging_vdkprototypes"
+
+        res = self._run_job(
+            "load_dimension_scd1_template_job",
+            {
+                "source_schema": test_schema,
+                "source_view": source_view,
+                "target_schema": test_schema,
+                "target_table": target_table,
+                "check": "use_positive_check",
+                "staging_schema": staging_schema,
+            },
+        )
+
+        assert not res.exception
+        actual_rs = self._run_query(f"SELECT * FROM {test_schema}.{target_table}")
+        expected_rs = self._run_query(f"SELECT * FROM {test_schema}.{source_view}")
+        assert actual_rs.output and expected_rs.output
+        assert actual_rs.output == expected_rs.output
+
+    def test_load_dimension_scd1_checks_negative(self) -> None:
+
+        test_schema = "vdkprototypes"
+        source_view = "vw_dim_org"
+        target_table = "dw_dim_org"
```

Comment:
To avoid situaitons in which another test populates the data in a table. Let's use distinct names

```
source_view="vw_dim_org_check_negative"
target_table ="dw_dim_org_check_negative"

```

Reply:
Done